production/ksonnet/README.md

```diff
@@ -39,6 +39,7 @@ local promtail = import 'promtail/promtail.libsonnet';
 promtail + {
   _config+:: {
     namespace: 'loki',
+    dataroot: '/var/lib/docker',
```

Comment:
I think this setting should be moved to inside the `promtail_config` block.

Reply:
sure, all done.